blog/2024-10-30-agama-10.mdx

```diff
@@ -0,0 +1,199 @@
+---
+date: 2024-10-30 10:00:00 +00:00
+title: Releasing version 10
+description: Summary of the Agama 10 release
+tags:
+  - release
+---
+
+As you can see we have a new place for our Agama blog! So let's start with announcing the Agama
+version 10 release at this new place! :tada:
+
+{/* truncate */}
+
+We released the Agama version 10 quite some time ago, but we did not announce it yet because of lack
+of time. So if you downloaded the Agama installer recently then you are already using this new
+version.
+
+We decided to separate the Agama blog posts from the YaST blog as those tools are quite different
+and to avoid confusion about what is what in the future. So welcome at the new blogging place for
+Agama!
+
+## QR codes with Agama URL {#qr}
+
+Let's start with some cool and fancy stuff! :smiley:
+
+We found out that it is possible to display QR code even on a Linux text console. As QR codes can
+contain an URL then why not use it also for Agama? If you use a mobile phone for connecting to the
+running Agama instance this makes the process much easier.
+
+![Linux console with Agama QR code](../static/img/blog/2024-10-30/qr-code.png)
+
+The QR code is not displayed on standard 80x25 or 80x24 terminals because there is not enough space,
+it is displayed only on bigger screens.
+
+## Syntax highlighting {#highlighting}
+
+You can edit the current Agama configuration in a text editor using the `agama config edit` command.
+This opens a text editor (Vim by default) with the current Agama configuration in the JSON format.
+
+Previously the configuration was displayed as a plain text file without any syntax highlighting. Now
+you can enjoy JSON syntax highlighting which makes editing a bit more pleasant.
+
+import SmallImageBox from "@site/src/components/SmallImageBox";
+
+<SmallImageBox>
+  ![Editor without syntax highlighting](../static/img/blog/2024-10-30/config-edit-before.png)
+  ![Editor with syntax highlighting](../static/img/blog/2024-10-30/config-edit-after.png)
+</SmallImageBox>
+
+You can use any another text editor for editing the configuration but as the Live ISO content is
+limited it only offers Joe as an alternative.
+
+```bash
+EDITOR=joe agama config edit
+```
+
+:::tip
+
+You can install more editors using the usual `zypper install` command. But be careful, see the
+<a href="#free-space-warning">warning</a> below.
+
+:::
+
+## Product selection {#products}
+
+We changed a bit the list of products which can be installed with Agama. Since version 10 it is
+possible to install openSUSE Leap 16.0 Alpha. We also created a SLES 16 flavor but for now it is not
+available for public.
+
+The product selection dialog now contains the product icons so the product selection should be more
```

Comment:
I'm sorry for being late on adding this tiny NP: I'd say logos, not icons. Just in case we can edit the blog entries without too much pain.